spec.html

```diff
@@ -50790,11 +50786,11 @@ <h1>String.prototype.fixed ( )</h1>
       </emu-annex>
 
       <emu-annex id="sec-string.prototype.fontcolor">
-        <h1>String.prototype.fontcolor ( _color_ )</h1>
+        <h1>String.prototype.fontcolor ( _colour_ )</h1>
```

Comment:
I hate this very much but I guess it's fine.

Reply:
the argument has always been named "color", imo this isn't just spec-internal and shouldn't be changed.

Reply:
Yeah we were back and forth on this one. Definitely open to changing it back in a follow-up, but the PR was in a good-enough state for now.